initScripts/aarch64/Ubuntu_16.04/Docker_17.06.sh

```diff
@@ -13,8 +13,7 @@ export BASE_UUID="$(cat /proc/sys/kernel/random/uuid)"
 export BASE_DIR="$SHIPPABLE_RUNTIME_DIR/$BASE_UUID"
 export REQPROC_DIR="$BASE_DIR/reqProc"
 export REQEXEC_DIR="$BASE_DIR/reqExec"
-export REQEXEC_BIN_DIR="$BASE_DIR/reqExec/bin"
-export REQEXEC_BIN_PATH="$REQEXEC_BIN_DIR/dist/main/main"
+export REQEXEC_BIN_PATH="$REQEXEC_DIR/$NODE_ARCHITECTURE/$NODE_OPERATING_SYSTEM/dist/main/main"
```

Comment:
@scriptnull please note this change. When building the reqProc image, this should be the path to the binary.